server/Main.hs

```diff
@@ -335,13 +354,16 @@ main = do
                      { corsRequestHeaders = ["Content-Type", "Authorization"]
                      , corsMethods = [methodGet,methodPost,methodPut,methodDelete,methodHead]
                      }
+      -- get the whitelisted addresses from $WLIST evn var
+      -- if useWhitelist is set to true, then the whitelist is ignored
```

Comment:
Shouldn't the comment be "if useWhitelist is set to false, the the whitelist is ignored?"

Reply:
done